docs/DEPLOYMENT.md

```diff
@@ -20,27 +23,99 @@
 - Kubernetes Cluster
   - You can also deploy on [Minkube][minikube], [kind][kind] or [OpenShift][openshift].
 - [kubectl][kubectl]
-- [ko][ko]
+- [docker][docker] or [podman][podman]
 
-  - You can find installation steps [here][ko] or if you have Go installed then you can execute the command :-
+## Build Images
 
-    ```bash
-    go get github.com/google/ko/cmd/ko
-    ```
+Lets build and push all images to a registry and then create deployments.
 
-- [docker][docker] or [podman][podman]
+### UI Image
 
-## Deploy API Service
+Ensure you are in `ui` directory and logged in you image registry.
+
+```
+docker build -t <image> . && docker push <image>
+```
+
+Replace `<image>` with the registry and image name. 
+eg. `quay.io/<username>/ui`
+
+### API & DB Migration Image
+
+Ensure you are in `api` directory and logged in you image registry.
+
+Build the API image using below command
+```
+docker build -t <image> . && docker push <image>
```

Comment:
I think one needs to tag the image before pushing

Reply:
`<image>` is a generic thing which can include both image name and tag right?